crates/forge/tests/integration/resources.rs

```diff
@@ -0,0 +1,195 @@
+use blockifier::execution::deprecated_syscalls::DeprecatedSyscallSelector::{
+    Deploy, EmitEvent, GetBlockHash, GetExecutionInfo, Keccak, SendMessageToL1, StorageRead,
+    StorageWrite,
+};
+use indoc::indoc;
+use std::path::Path;
+use test_utils::runner::Contract;
+use test_utils::running_tests::run_test_case;
+use test_utils::{assert_builtins, assert_passed, assert_syscalls, test_case};
+
+#[test]
+fn builtins_count() {
+    let test = test_case!(indoc!(
+        r"
+            #[test]
+            fn range_check() {
+                assert(1_u8 >= 1_u8, 'error message');
+            }
+
+            #[test]
+            fn bitwise() {
+                let _bitwise = 1_u8 & 1_u8;
+                assert(1 == 1, 'error message');
+            }
+
+            #[test]
+            fn pedersen() {
+                core::pedersen::pedersen(1, 2);
+                assert(1 == 1, 'error message');
+            }
+
+            #[test]
+            fn poseidon() {
+                core::poseidon::hades_permutation(0, 0, 0);
+                assert(1 == 1, 'error message');
+            }
+
+            #[test]
+            fn ec_op() {
+                let ec_point = core::ec::EcPointTrait::new_from_x(1).unwrap();
+                core::ec::EcPointTrait::mul(ec_point, 2);
+                assert(1 == 1, 'error message');
+            }
+        "
+    ));
+
+    let result = run_test_case(&test);
+
+    assert_passed!(result);
+
+    // No ECDSA and Keccak builtins
```

Comment:
Why not? 🤔

Reply:
I haven't found (and neither did @war-in when implementing gas) any way to make these builtins execute. Keccak is counted as a syscall in the test below though.